zstacklib/zstacklib/utils/gpu.py

```
@@ -73,6 +73,31 @@ def pre_detach_from_host(vendor):
     return 0, None
 
 
+def get_gpu_device_vendor(pci_device_address):
+    r, o, e = bash_roe("lspci -s %s" % pci_device_address)
```

Ruff (0.14.6)

Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)